### PR TITLE
Removes `PolicyManager` and `StakingInterface` contracts

### PR DESCRIPTION
**Pull request by vzotova**
**Type of PR:**
- Other

**Required reviews:** 
- 2

**What this does:**
Removes `PolicyManager` and `StakingInterface` contracts/agents/deployers and related tests

### CODE REVIEW

**Review by KPrasch — APPROVED** [review]

File: nucypher/characters/unlawful.py [hunk 1]
```diff
@@ -159,29 +159,3 @@ def circumvent_safegaurds_and_grant_without_paying(self, *args, **kwargs):
         """
         with patch("nucypher.policy.policies.Policy._publish", self.grant_without_paying):
             return self.grant_without_paying(*args, **kwargs)
-
-    def grant_while_paying_the_wrong_nodes(self,
```
Comment (derekpierre) [inline nucypher/characters/unlawful.py:163]: Possible to reuse this test with SubscriptionManager?...maybe not.

**Review by derekpierre — APPROVED** [review]
🎸

File: nucypher/characters/unlawful.py (hunk 1 above)
Comment (KPrasch) [reply nucypher/characters/unlawful.py:163]: SubscriptionManager facilitates payment on a network-wide basis and Alice no longer pays specific nodes.  The is another test in this module that does test for a sinpa attack against `SubscriptionManager`.

Comment (derekpierre) [reply nucypher/characters/unlawful.py:163]: 👍

**Pull request merged**